### PR TITLE
Setup GitHub Actions to build Node.js project on PRs

### PR DESCRIPTION
**Pull request by ogeeDeveloper**
This pull request introduces a new GitHub Actions workflow designed to automate the build process for our Node.js project whenever new pull requests are made. The workflow is defined in the .github/workflows/nodejs_build.yml file and contains the following steps:

Setup: The workflow starts by setting up the Node.js environment using the latest stable version.
Dependency Installation: It then proceeds to install all the necessary project dependencies using npm ci to ensure a clean, reliable installation.
Build: Finally, it executes npm run build to build the project, ensuring that no integration issues are introduced.
This automated process aims to detect any potential problems early in the development cycle, preventing integration issues from reaching the main branch. The workflow runs on the latest Ubuntu virtual environment and is triggered by any new PR made against the main branch.

### CODE REVIEW

File: .github/workflows/nodejs_build.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```
Comment (coltonehrman) [inline .github/workflows/nodejs_build.yml:13]: Let's use only `20.x` for now

Comment (ogeeDeveloper) [reply .github/workflows/nodejs_build.yml:13]: Modifying it now

Comment (ogeeDeveloper) [reply .github/workflows/nodejs_build.yml]: The change was applied to the node version

**Review by coltonehrman — APPROVED** [review]
Awesome! :D

**Pull request merged**